medusa/clients/torrent/utorrent_client.py

```diff
@@ -52,13 +52,14 @@ def _request(self, method='get', params=None, data=None, files=None, cookies=Non
     def _get_auth(self):
 
         self.response = self.session.get(urljoin(self.url, 'token.html'), verify=False)
-        self.auth = re.findall('<div.*?>(.*?)</', self.response.text)[0]
-
-        return self.auth if not self.response.status_code == 404 else None
+        if not self.response.status_code == 404:
+            self.auth = re.findall('<div.*?>(.*?)</', self.response.text)[0]
+            return self.auth
 
     def _add_torrent_uri(self, result):
         return self._request(params={
             'action': 'add-url',
+            # requests (?) limits the param length to 1024 chars
             's': result.url[:1024],
```

Comment:
Actually how that I think this may be problematic...if the dn param + the hash param exceeds the 1024 by themselves, meaning it will become a tracker less torrent and will only work the client has DHT/PEX enabled.
Did you understood @medariox ?